plugins/@grouparoo/airtable/src/lib/destination/exportArrayProperties.ts

```diff
@@ -0,0 +1,19 @@
+import { ExportArrayPropertiesMethod } from "@grouparoo/core";
```

Comment:
This may be one of the first destinations that supports array properties in a meaningful way!